File: tools/tailor_cv_py/src/tailor_cv/reader.py
from pathlib import Path


def read_file(path: str) -> str:
    p = Path(path)
    if not p.exists():
        raise FileNotFoundError(f"File not found: {path}")
    content = p.read_text(encoding="utf-8").strip()
    if not content:
        raise ValueError(f"File is empty: {path}")
    return content

File: tools/tailor_cv_ts/src/reader.ts
import { existsSync, readFileSync } from "fs";

export function readFile(path: string): string {
  if (!existsSync(path)) {
    throw new Error(`File not found: ${path}`);
  }
  const content = readFileSync(path, "utf-8").trim();
  if (!content) {
    throw new Error(`File is empty: ${path}`);
  }
  return content;
}
